app/factory/Layer.js

```diff
@@ -216,8 +216,13 @@ Ext.define('CpsiMapview.factory.Layer', {
         var styles = layerConf.serverOptions.styles;
         var activatedStyle;
         if (Ext.isArray(layerConf.styles) && layerConf.styles.length) {
-            styles = layerConf.styles[0];
-            activatedStyle = layerConf.styles[0];
+            // check if first possible SLD style list is an object (with STYLES
+            // name and UI alias) or if the STYLES name is directly provided.
+            var firstStyle = layerConf.styles[0];
+            var stylesWmsParam =
+                Ext.isObject(firstStyle) ? firstStyle.name : firstStyle[0];
```

Comment:
One observation: here you call `firstStyle[0]` whereas in the other edits (WFS and VT) you only call `firstStyle`. Just wondering if this intentionally

Reply:
Good catch, I fixed and rebased...thanks